power_grid_model_c/power_grid_model/include/power_grid_model/timer.hpp

```diff
@@ -57,7 +57,7 @@ class Timer {
         if (info_ != nullptr) {
             auto const now = Clock::now();
             auto const duration = Duration(now - start_);
-            info_->operator[](Timer::make_key(code_, name_)) += (double)duration.count();
+            info_->operator[](Timer::make_key(code_, name_)) += duration.count();
```

Comment:
please double-check that this indeed results in exactly the same behaviour, and that no compiler/OS-dependency is introduced